Library/Formula/fakesmtp.rb

```diff
@@ -0,0 +1,53 @@
+class Fakesmtp < Formula
+  desc "Dummy SMTP server with GUI for testing emails in applications easily"
+  homepage "https://nilhcem.github.io/FakeSMTP/"
+  url "https://github.com/Nilhcem/FakeSMTP/archive/v2.0.tar.gz"
+  sha256 "8cb623f32b55f814af43b731432eb14da80d95cb13059c79423265037cda9dfb"
+
+  depends_on :java
+  depends_on "maven" => :build
+
+  def install
+    ENV.java_cache
+    system "mvn", "package", "-Dmaven.test.skip=true"
+    libexec.install "target/fakeSMTP-#{version}.jar"
+    (bin/"fakesmtp").write <<-EOS.undent
+      #!/bin/bash
+
+      args+=()
+
+      for i in "$@"; do
+        case $i in
+          -b|--background) args+=('-Djava.awt.headless=true');;
+        esac
+      done
+
+      exec java ${args[@]} -jar #{libexec}/fakeSMTP-#{version}.jar "$@"
+      EOS
+  end
+
+  test do
+    require "net/smtp"
+
+    pid = fork do
+      exec bin/"fakesmtp", "-b", "-s", "-p2025"
```

Comment:
Well, `fakeSMTP` hast the option to run without a GUI.

```
 -b,--background            If specified, does not start the GUI. Must be
                            used with the -s (--start-server) argument
```

But without the `java` parameter `-Djava.awt.headless=true` it quickly shows a spashscreen on start. This happends even if the `--background` flag is set. This prevents `fakeSMTP` to run on headless servers.

This would make the test fail on travis ci and jenkins severer.

Therefore I added the custom jar script, which adds the `-Djava.awt.headless=true` flag if the user attempts to run `fakeSTMP` without a GUI.